url.bs

```diff
@@ -3181,11 +3169,9 @@ console.log(url.searchParams.get('b')); // "~"</code></pre>
  <!-- From https://url.spec.whatwg.org/#concept-urlencoded-byte-serializer, inverted.
       Do not change this without double checking URLENCODED-UNITS -->
 
- <p>Ignoring encodings (use <a>UTF-8</a>), {{URL/search}} will percent-encode U+0000 NULL to
- U+0020 SPACE, inclusive, U+0022 ("), U+0023 (#), U+0027 (') varying on <a>is special</a>,
- U+003C (&lt;), U+003E (>), and anything greater than U+007E (~).
- <!-- From https://url.spec.whatwg.org/#query-state.
-      Do not change this without double checking QUERY-UNITS -->
+ <p>Ignoring encodings (use <a>UTF-8</a>), {{URL/search}} will percent-encode anything in the
+ <a>query percent-encode set</a> or, varying on <a>is special</a>, the
+ <a>special-query percent-encode set</a>.
```

Comment:
Maybe "will percent-encode anything in either the query percent-encode set or the special-query percent-encode set (depending on whether or not the URL is special)".